internal/build/build.go

```diff
@@ -175,3 +177,51 @@ func randString(n int) string {
 	}
 	return string(b)
 }
+
+func (l *Lifecycle) CopyApp(ctx context.Context, containerID string) error {
```

Comment:
It feels weird to me that `CopyApp` lives in build.go and `WriteStackToml` lives in phases.go - could they be consolidated?

Reply:
Yeah, I didn't give the organization too much as it seemed disorganized before I got here and wanted to limit the number of changes. I'll see what I can do with more thought.